CHANGES.md

```diff
@@ -6,6 +6,8 @@
 
 - Added `FeatureDetection.supportsBigInt64Array`, `FeatureDetection.supportsBigUint64Array` and `FeatureDetection.supportsBigInt`.
 
+- Added `ShadowMap.fadingEnabled` to control whether shadows fade out when the light source is close to the horizon. [#9565](https://github.com/CesiumGS/cesium/pull/9565)
```

Comment:
@kalosma My mistake; I missed this the first time - to be consistent with other changes could you reword this as: 
```suggestion
- Added `options.fadingEnabled` parameter to `ShadowMap` to control whether shadows fade out when the light source is close to the horizon. [#9565](https://github.com/CesiumGS/cesium/pull/9565)
```

Otherwise this is good to go; I'll merge once that's done. Thanks!